### DISCUSSION TITLE
Add new filters for the LLM as a Judge Evaluation (other scores and cost)

### DISCUSSION THREAD

**Discussion by laisalvescw** [langfuse/langfuse#4106] [Ideas]
### Describe the feature or potential improvement

It would be great to have the same filtering options available in the Traces UI within the LLM as a Judge, as some options currently seem to be missing, such as "total_cost" or "scores"

<img width="1256" alt="image" src="https://github.com/user-attachments/assets/ab2b283f-6e2b-4582-8591-1feeafb0b5e2">

**Comment by marcklingen** [comment]
thanks for sharing!

**Comment by Lotte-Verheyden** [comment]
This would solve https://github.com/orgs/langfuse/discussions/11994#discussioncomment-15785273 too
